packages/server/src/tests/epics/momentumTest.ts

```diff
@@ -86,13 +87,39 @@ export function isIncreasedEngagement(
     const { categoryForThirdMonth, categoryForSecondMonth, categoryForFirstMonth } =
         getCategoriesForThreeMonths(articleHistory, now);
 
+    const articleHistoryWithoutTags = articleHistory.map(({ week, count }) => ({
```

Comment:
is `articleHistory` potentially 52 elements?

Reply:
we've already cut it down to 13 elements